frontend/test/projects/projects.sagas.spec.ts

```diff
@@ -20,11 +20,29 @@ import { expectSaga } from 'redux-saga-test-plan';
 import * as ProjectsSaga from '@/v5/store/projects/projects.sagas';
 import { ProjectsActions } from '@/v5/store/projects/projects.redux';
 import { mockServer } from '../../internals/testing/mockServer';
-import { alertAction } from '../test.helpers';
+import { alertAction, createTestStore } from '../test.helpers';
+import { TeamspacesActions } from '@/v5/store/teamspaces/teamspaces.redux';
+import { projectMockFactory } from './projects.fixtures';
+import { DialogsTypes } from '@/v5/store/dialogs/dialogs.redux';
+import { templateMockFactory } from '../tickets/tickets.fixture';
+import { selectCurrentProjectTemplates } from '@/v5/store/projects/projects.selectors';
+import { FederationsActions } from '@/v5/store/federations/federations.redux';
+import { federationMockFactory } from '../federations/federations.fixtures';
 
 describe('Teamspaces: sagas', () => {
```

Comment:
I thought you changed this to 'Projects: sagas' or was that in another PR?

Reply:
It was indeed this one, but I forgot to fix it. Will do for the next review